README.md

```diff
@@ -0,0 +1,19 @@
+CHT Pipeline
+========================
+
+CHT Pipeline is a tool used to define data models for transforming the raw data we get from Couch DB into models that can then be queried to build dashboards
+
+# Installation
```

Comment:
Let's be more opinionated here. If these are equivalent then pick the best/easiest option. If they're not equivalent then let's document why you should pick one over the other.

Also, is this dev environment or production installation? Let's be clear.